api/server/sdk/volume_ops.go

```diff
@@ -42,12 +101,26 @@ func (s *VolumeServer) create(
 	// Check if the volume has already been created or is in process of creation
 	volName := locator.GetName()
 	v, err := util.VolumeFromName(s.driver(ctx), volName)
-	if err == nil {
+
+	// If the volume is still there but it is being delete, then wait until it is removed
+	if err == nil && v.GetState() == api.VolumeState_VOLUME_STATE_DELETED {
+		if err = s.waitForVolumeRemoved(ctx, volName); err != nil {
+			return "", err
```

Comment:
can we throw a more user friendly error message here? Looks like currently it will throw a generic timed out error message from the util package. Instead it should be something along the lines

```
volume with same name is in the process of being deleted. Timed out waiting for deletion to complete.
```